server/handler/base.go

```diff
@@ -193,5 +206,58 @@ func (b *Base) EvaluateFetchedConfig(ctx context.Context, prctx pull.Context, cl
 		return errors.Errorf("evaluation resulted in unexpected state: %s", result.Status)
 	}
 
+	if statusState == "pending" && prctx.IsDraft() != nil && !*prctx.IsDraft() {
+		subsetCurrentReviewers, _, err := client.PullRequests.ListReviewers(ctx, prctx.RepositoryOwner(), prctx.RepositoryName(), prctx.Number(), &github.ListOptions{
+			Page:    0,
+			PerPage: 10,
+		})
+
+		if err != nil {
+			logger.Warn().Err(err).Msg("Unable to list request reviewers")
+		}
+
+		if subsetCurrentReviewers != nil && len(subsetCurrentReviewers.Users) == 0 && len(subsetCurrentReviewers.Teams) == 0 {
+			// look for requested reviewers
+			leafNodes := findLeafChildren(result)
+			var requestedUsers, requestedTeams []string
+			for _, child := range leafNodes {
+				if child.Error != nil {
```

Comment:
We only care about nodes that are pending. Maybe `findLeafChildren` could also filter by state (and not having an error)?